src/app/ui/editor/select_box_state.cpp

```diff
@@ -29,12 +30,16 @@ using namespace ui;
 
 SelectBoxState::SelectBoxState(SelectBoxDelegate* delegate, const gfx::Rect& rc, Flags flags)
   : m_delegate(delegate)
-  , m_rulers(4)
+  , m_rulers((int(flags) & int(Flags::PaddingRulers))? 6: 4)
   , m_rulersDragAlign(0)
   , m_selectingBox(false)
   , m_flags(flags)
 {
   setBoxBounds(rc);
+  if (hasFlag(Flags::PaddingRulers)) {
+    gfx::SizeT<int> padding(0, 0);
```

Comment:
```suggestion
    gfx::Size padding(0, 0);
```
As we talked, we can use `gfx::Size` directly here.